src/shared/helpers.ts

```diff
@@ -118,8 +118,33 @@ export const rotateTicket = async (ticket: string): Promise<string> => {
   return new_ticket
 }
 
-export const getPermissionVariablesFromCookie = (req: RequestExtended): PermissionVariables => {
-  const { permission_variables } = COOKIES.SECRET ? req.signedCookies : req.cookies
-  if (!permission_variables) throw new Error('No permission variables')
-  return JSON.parse(permission_variables)
+export function newRefreshExpiry(): number {
+  const now = new Date()
+  const days = JWT.REFRESH_EXPIRES_IN / 1440
```

Comment:
Could we add a comment about the `1440` here to increase the code readability?

ex: // converting from minutes to days